projects/canopy/src/lib/card/card.component.spec.ts

```diff
@@ -92,15 +99,41 @@ describe('LgCardComponent', () => {
     });
 
     it('should expect card header to render', () => {
-      expect(
-        fixture.debugElement.query(By.directive(LgCardHeaderComponent)),
-      ).toBeDefined();
+      expect(debugElement.query(By.directive(LgCardHeaderComponent))).toBeTruthy();
     });
 
     it('should expect card content to render', () => {
+      expect(debugElement.query(By.directive(LgCardContentComponent))).toBeTruthy();
+    });
+  });
+
+  describe('when there is lg-card-hero-img and lg-card-content', () => {
+    beforeEach(() => {
+      const localFixture = MockRender(`
+        <lg-card>
+         <lg-card-hero-img imageUrl="/test"></lg-card-hero-img>
```

Comment:
Not sure how these tests pass since `imageUrl` is not a thing anymore, right?

Reply:
Weirdly enough, it was adding ``imageUrl`` as a attribute on that element 🤔 
I've rewrote the test to check for the ``src`` property on the component instance.